modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -144,6 +144,12 @@ public FileEntry addAttachment(
 			false);
 	}
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
+	 * #addKBArticle(String, long, long, long, String, String, String, String,
+	 * String, String[], String[], ServiceContext)}
+	 */
+	@Deprecated
```

Comment:
I've seen in a Lima team PR, that they also find the usages of the deprecated methods and replace them using the new method with externalReferenceCode passing null. This could be a good practice to follow as it will be safer when we removed the deprecated methods

Reply:
Yeah! We could do that